backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -61,28 +57,18 @@ protected BaseTestInfo() {
 
   protected BaseTestInfo(BaseTestInfo orig) {
     this(orig.getPatient(), orig.getFacility());
-    if (orig.getDeviceSpecimen() != null) {
-      this.setDeviceSpecimen(orig.getDeviceSpecimen());
-    } else {
-      this.setDeviceTypeAndSpecimenType(orig.getDeviceType(), orig.getSpecimenType());
-    }
   }
 
-  protected BaseTestInfo(Person patient, Facility facility, DeviceSpecimenType deviceSpecimen) {
+  protected BaseTestInfo(Person patient, Facility facility) {
     super();
     this.patient = patient;
     this.facility = facility;
     this.organization = facility.getOrganization();
-    this.deviceSpecimen = deviceSpecimen;
-    this.deviceType = deviceSpecimen.getDeviceType();
-    this.specimenType = deviceSpecimen.getSpecimenType();
+    this.deviceType = facility.getDefaultDeviceType();
+    this.specimenType = facility.getDefaultSpecimenType();
```

Comment:
Just to make sure I understand this correctly - the way this should work is that a TestOrder/TestEvent will be created in TestOrderService and initialized with the facility default device and specimen type, then overwritten with calls to `setDeviceTypeAndSpecimenType`. Is that right?

Reply:
That us correct, currently its using the [`facility.getDefaultDeviceSpecimen()`](https://github.com/CDCgov/prime-simplereport/pull/5012/files/f4fe6bd1ebe621ce601d645f0cfd19687a246799#diff-bcfa7cfe8b9e6655d7c3df481758105ff21c9a4ca3224b18e26a9cfaae9feac0L82),
 and this will change it to use the `facility.getDefaultDeviceType()` and `facility.getDefaultSpecimenType()`